sample_apps/rust/examples/utils/timestream_helper.rs

```diff
@@ -0,0 +1,136 @@
+use aws_sdk_timestreamwrite as timestream_write;
+use aws_types::region::Region;
+use clap::Parser;
+
+static DEFAULT_DATABASE_NAME: &str = "devops_multi_sample_application";
+static DEFAULT_REGION: &str = "us-east-1";
+static DEFAULT_TABLE_NAME: &str = "host_metrics_sample_application";
+
+#[derive(Parser, Debug)]
+#[command(version, about, long_about = None)]
+pub struct Args {
+    // The Timestream for LiveAnalytics database name to use for all queries
+    #[arg(short, long, default_value = DEFAULT_DATABASE_NAME)]
+    pub database_name: String,
+
+    // The name of the AWS region to use for queries
+    #[arg(short, long, default_value = DEFAULT_REGION)]
+    pub region: String,
+
+    // The Timestream for LiveAnalytics table name to use for all queries
+    #[arg(short, long, default_value = DEFAULT_TABLE_NAME)]
+    pub table_name: String,
+}
+
+pub async fn get_connection(
+    region: &String,
+) -> Result<timestream_write::Client, timestream_write::Error> {
+    let config = aws_config::defaults(aws_config::BehaviorVersion::latest())
+        .region(Region::new(region.to_owned()))
+        .load()
+        .await;
+    let (client, reload) = timestream_write::Client::new(&config)
+        .with_endpoint_discovery_enabled()
+        .await
+        .expect("Failed to get connection to Timestream");
+
+    tokio::task::spawn(reload.reload_task());
+    Ok(client)
+}
+
+pub async fn create_database(args: &Args) -> Result<(), timestream_write::Error> {
```

Comment:
Maybe `create_database` and `create_table` should accept a `String` instead of an `&Arg`. That way these functions can be reused in places where no args have been provided.

Reply:
Agreed, this is better.